packages/spor-react/src/datepicker/DatePicker.tsx

```diff
@@ -132,9 +135,9 @@ export function DatePicker({
           {state.isOpen && !props.isDisabled && (
             <Portal>
               <PopoverContent
-                backgroundColor="white"
                 color="darkGrey"
                 boxShadow="md"
+                sx={styles.calendar}
               >
                 <PopoverArrow backgroundColor="white" />
```

Comment:
![image](https://github.com/nsbno/spor/assets/98820614/331cbd8d-7bf4-47bf-91ad-0f518f5a1dce)
Hvordan får jeg `<PopoverArrow />` til å ta hensyn til darkmode her? Det fungerer ikke med sx-propen. Tror det har noe å gjøre med at dette allerede er en overskriving av stylingen i `popover.ts` 🤔

Reply:
Så jeg sliter med å finne de riktige trylleordene, men variabelen for å sette bakgrunnsfargen til popover-arrow er:

`--popper-arrow-bg`; og med litt fiddling så finner du kanskje ut av det.

```typescript
const $arrowBackground = cssVar("popper-arrow-bg");


const config = helpers.defineMultiStyleConfig({
    /** stuff  **/
    arrow: {
      [$arrowBackground.variable]: `colors.dimGrey`,  
});
```

Hva som er riktig parameter, som endrer seg i henhold til theming, er jeg litt usikker på. `dimGrey` er selvsagt ikke riktig.

Reply:
Nå som jeg priker i det så fant jeg ut at det fort blir noe sånt som:

```
  baseStyle: ({ colorMode }) => ({
      arrow: {
           [$arrowBackground.variable]: mode("colors.white", "colors.night")({ colorMode }),
      }
```